project/Dependencies.scala

```diff
@@ -7,14 +7,13 @@ object Dependencies {
     val ProtocolVersionMajor = 1
     val ProtocolVersionMinor = 1
   }
-  val AkkaRuntimeVersion = sys.props.getOrElse("akka-runtime.version", "1.5.22")
+  val AkkaRuntimeVersion = sys.props.getOrElse("akka-runtime.version", "1.5.23")
```

Comment:
Strange, the diff is wrong, because 1.5.27 is the current version in the main branch